examples/js/lines/LineMaterial.js

```diff
@@ -272,6 +274,24 @@ THREE.LineMaterial = function ( parameters ) {
 
 		},
 
+		screenSpaceWidth: {
+
+			enumerable: true,
+
+			get: function () {
+
+				return this.uniforms.screenSpaceWidth.value === 1;
```

Comment:
Probably, users would expect a non-boolean value.

Reply:
It's important that this be `0` or `1` so a boolean value made sense and [sizeAttenuation](https://threejs.org/docs/#api/en/materials/PointsMaterial.sizeAttenuation) on PointsMaterial, which is a similar toggle for the shader, is a boolean, as well.

I see that the name `screenSpaceWidth` may be a bit confusing, though, and imply that this defines what the screen space width should be -- not whether the width should be calculated in screen space. Maybe another name would be better? `sizeAttenuation` didn't quite feel right but at least it would be consistent?

Reply:
Oh, I see. Maybe: 'isAttenuated'?
Also if it is a boolean value, then it is better to use ?: operator instead of the "mix" function for performance reasons.

Reply:
My understanding has always been that [conditional statements can hinder performance depending on the graphics hardware used](https://stackoverflow.com/questions/37827216/does-if-statements-slow-down-my-shader) so I've often opted to use something like `mix` in simple cases but I can change it if the maintainers prefer a condition.

Reply:
1) ?: is easier to read than mix
2) ?: operator would execute both "branches" as `mix`. But ?: operator masks a "wrong" branch and takes a "right" branch. `mix` have to perform arithmetic operations in case of compiler is not smart enough.
3) As said in the answer from Stackoverflow "statically uniform branching" is fast on modern HW.
4) Compiler knows that it can replace ?: by `mix` for performance reasons, the opposite is not always true.